projects/packages/backup/src/js/components/backup-storage-space/storage-help-popover/style.scss

```diff
@@ -0,0 +1,89 @@
+.backup-storage-space {
+	
+    &__help-popover {
```

Comment:
Sorry for nit 😅
It seems this block has been indented with white spaces instead of tabs. It would be great to double-check the indentations of this file.